packages/lib/passport-interface/src/EncryptedStorage.ts

```diff
@@ -129,7 +162,13 @@ export async function deserializeStorage(
   let pcds: PCDCollection;
   let subscriptions: FeedSubscriptionManager;
 
-  if (isSyncedEncryptedStorageV4(storage)) {
+  if (isSyncedEncryptedStorageV5(storage)) {
+    pcds = await PCDCollection.deserialize(pcdPackages, storage.pcds);
+    subscriptions = FeedSubscriptionManager.deserialize(
+      new NetworkFeedApi(),
+      storage.subscriptions
+    );
+  } else if (isSyncedEncryptedStorageV4(storage)) {
```

Comment:
Not unreasonable, though it's confusing in its own way as I said.
I feel like if this list of versions is going to grow, we might want to restructure it as a series of migrations which run sequentially, rather than individual upgrades from each version to latest.  That would've made it clear that the migration steps from 4 to 5 were empty.